src/utils.js

```diff
@@ -49,10 +49,12 @@ function safeCompare (item1, conditional, item2) {
 }
 
 // Get creation time of a Discord snowflake
+const Moment = require('moment')
```

Comment:
Non-constructor methods should use lowercase. This should also be moved to the top level, beneath the eslint-disable comment for the sake of consistency.